app/models/mongo/StateBenefitsUserData.scala

```diff
@@ -31,13 +32,15 @@ case class StateBenefitsUserData(benefitType: String,
                                  mtdItId: String,
                                  nino: String,
                                  taxYear: Int,
-                                 isPriorSubmission: Boolean,
+                                 benefitDataType: String,
                                  claim: Option[ClaimCYAModel],
                                  lastUpdated: LocalDateTime = LocalDateTime.now(ZoneOffset.UTC)) {
 
-  lazy val isHmrcData: Boolean = claim.exists(_.isHmrcData)
-
-  lazy val isNewClaim: Boolean = claim.exists(_.benefitId.isEmpty)
+  lazy val isPriorSubmission: Boolean = claim.exists(_.benefitId.isDefined)
+  lazy val isNewClaim: Boolean = !isPriorSubmission
```

Comment:
Do we need to have both `isPriorSubmission` and `isNewClaim`?

Reply:
If you are in favor to use positives when we do If clauses, than Yes :)